### PR TITLE
[*] Resources Getter - Support MSSQL records list retrieval without order in the query

### PR DESCRIPTION
**Pull request by lucasscariot**
## Pull Request checklist:

- [x] Write an explicit title for the Pull Request
- [x] Write changes made in the CHANGELOG.md
- [x] Test manually the implemented changes
- [x] Review my own code (indentation, syntax, style, simplicity, readability)
- [x] Wonder if you can improve the existing code

### CODE REVIEW

**Comment by lucasscariot** [comment]
[(?) Liana - MSSQL pagination does not seem to work properly on MSSQL 2014](https://trello.com/c/ufz2xB6x/1587-liana-mssql-pagination-does-not-seem-to-work-properly-on-mssql-2014)

**Comment by lucasscariot** [comment]
[(1) Liana - Collections sorted on primary key having belongsTo generate a broken SQL query.](https://trello.com/c/CGZKcp8Y/1610-liana-collections-sorted-on-primary-key-having-belongsto-generate-a-broken-sql-query)

**Comment by arnaudbesnier** [comment]
@lucasscariot tests on the HasMany added (the last one is broken and shows a regression with the refactoring)

File: services/query-builder.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    });
+
+    return includes;
+  };
+
+  this.getOrder = function () {
+    if (params.sort) {
+      var idField = _.keys(model.primaryKeys)[0];
+
+      if (Database.isMSSQL(opts) && params.sort.indexOf(idField) >= 0) {
```
Comment (arnaudbesnier) [inline services/query-builder.js]: A comment here to explain the edge case is really necessary

**Pull request merged**